main/src/ca/uwaterloo/flix/language/errors/EntryPointError.scala

```diff
@@ -222,4 +222,150 @@ object EntryPointError {
          |""".stripMargin
     }
   }
+
+  /**
+    * An error raised to indicate that the effect for default handler was not able to be found.
```

Comment:
not able to be found? Just not found?

But perhaps this needs to be reworded entirely. Something like: 

DefaultHandlerNotInEffectModule

with an error like "A default handler must be in the companion module of an effect."

Then expand with 

"Module Foo does not have a companion effect Foo" etc.